INSTALL.md

```diff
@@ -22,30 +22,12 @@ root# yum localinstall -y goldstone-server-{version}.rpm
 
 ## Install and configure Goldstone (as root)
 
-You have two ways to install Goldstone.
-
-### 1. Complete (requires terminal interaction)
-
 The installation script will check the Goldstone prerequisites. If all checks pass, it will then prompt you for additional configuration information. Execute these commands:
 
 ```bash
 root# cd /opt/goldstone
 root# . bin/activate
-root# fab full_install
-```
-
-### 2. Partial (requires no terminal interaction)
-
-**This section is a work in progress.**
-
-This installation requires no terminal interaction. But, you will have to perform additional installation steps (described below) after it completes.
-
-The installation script will check the Goldstone prerequisites. If all checks pass, it will then do a partial installation of Goldstone. Execute these commands, replacing "PASSWORD" with the password you want the Django administrative account to have:
-
-```bash
-root# cd /opt/goldstone
-root# . bin/activate
-root# fab install:django_admin_password=PASSWORD
+root# fab install
 ```
 
 #### Completing the installation
```

Comment:
Since fab install now does a full installation, The "Completing the installation" section should be deleted.  There's no need for the user to create entries using the client, right?

![screen shot 2015-06-12 at 8 11 47 am](https://cloud.githubusercontent.com/assets/231148/8133089/e559a1a2-10da-11e5-81d9-4c8624a3f502.png)